test/bandit/websocket/autobahn_test.exs

```diff
@@ -0,0 +1,109 @@
+defmodule WebsocketAutobahnTest do
+  use ExUnit.Case, async: false
+
+  @tag timeout: :infinity
+  @tag :autobahn
```

Comment:
This can likely take a lot of the config conventions from https://github.com/mtrudel/bandit/blob/main/test/bandit/http2/h2spec_test.exs as it's the same conceptual idea (albeit docker only). Specifically:

* Use the external_conformance tag for test inclusion (no test_helper or mix changes needed)
* Use capture_log to keep it quiet in the happy case. If the test fails, ex_unit will dump it all